BD-Automation-Engine/.env

```
@@ -0,0 +1,176 @@
+# Apify (for web scraping - ClearanceJobs, LinkedIn, competitor sites)
+APIFY_API_TOKEN=apify_api_Dn5VKCgHLqfT8L3zgdaBFPKUUNnryQ1qkJj6
+
+# ===========================================
+# NOTION INTEGRATION (PTS BD Intelligence System)
+# ===========================================
+
+# Notion API Token (from developers.notion.com)
+NOTION_TOKEN=ntn_R48446747822wNuLRcByvXQQDsGQy1K13e58Wcp4w4waRH
+
+# Notion MCP Server Headers (for @notionhq/notion-mcp-server)
+OPENAPI_MCP_HEADERS={"Authorization": "Bearer ntn_R48446747822wNuLRcByvXQQDsGQy1K13e58Wcp4w4waRH", "Notion-Version": "2022-06-28"}
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)